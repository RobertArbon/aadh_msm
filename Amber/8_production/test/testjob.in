AADH production trajectory 
&cntrl
  imin   = 0,                    ! INP Standard MD, no minimization 
  irest  = 1,                    ! INP Restart simulation
  ntx    = 5,                    ! INP Read in coordinates and velocities
  ntpr   = 5000,                ! OUT print frequencey for energy info to mdout
  ntwx   = 5000,                ! OUT write every 10ps
  ntwr   = 50000,               ! OUT write restart every 100ps 
  iwrap  = 1,                    ! OUT wrap coordinates for long trajectory
  ntc    = 2,                    ! RESTRAINTS SHAKE for TIP3P
  nscm   = 1000,                 ! MD frequency to remove C.o.M motion
  dt     = 0.002,                ! MD timestep 
  nrespa = 1,                    ! MD RESPA time step.  Keep nrespa * dt <= 4
!  nstlim = 50000000,             ! MD Number of time steps
 nstlim = 1000,             ! MD Number of time steps
  tempi  = 310.0,                ! TEMP initial temperature
  temp0  = 310.0,                ! TEMP final temperature
  ntt    = 3,                    ! TEMP Temp control 3 = Langevin, 0 = NVE
  gamma_ln = 5.0,                ! TEMP collision frequency
  ntf    = 2,                    ! POTENTIAL omit H-atoms in potential evaluation 
  ntb    = 1,                    ! POTENTIAL Constant Volume PBC for non-bonded forces
  cut    = 12.0,                 ! POTENTIAL Non-bonded cutoff
  fswitch = 10.0,                ! POTENTIAL Switching function start length 
/

