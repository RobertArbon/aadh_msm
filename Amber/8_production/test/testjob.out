
          -------------------------------------------------------
          Amber 16 PMEMD                              2016
          -------------------------------------------------------

| PMEMD implementation of SANDER, Release 16

| Run on 04/12/2017 at 13:03:17

|   Executable path: pmemd.cuda
| Working directory: /panfs/panasas01/chem/ra15808/aadh/md/Amber/8_production
|          Hostname: newblue4

  [-O]verwriting output

File Assignments:
|   MDIN: testjob.in                                                            
|  MDOUT: testjob.out                                                           
| INPCRD: ../common/equil6.rst                                                  
|   PARM: ../common/2agy_final_min.prmtop                                       
| RESTRT: testjob.rst                                                           
|   REFC: refc                                                                  
|  MDVEL: mdvel                                                                 
|   MDEN: mden                                                                  
|  MDCRD: testjob.mdcrd                                                         
| MDINFO: testjob.info                                                          
|  MDFRC: mdfrc                                                                 


 Here is the input file:

AADH production trajectory                                                     
&cntrl                                                                         
  imin   = 0,                    ! INP Standard MD, no minimization            
  irest  = 1,                    ! INP Restart simulation                      
  ntx    = 5,                    ! INP Read in coordinates and velocities      
  ntpr   = 5000,                ! OUT print frequencey for energy info to mdout
  ntwx   = 5000,                ! OUT write every 10ps                         
  ntwr   = 50000,               ! OUT write restart every 100ps                
  iwrap  = 1,                    ! OUT wrap coordinates for long trajectory    
  ntc    = 2,                    ! RESTRAINTS SHAKE for TIP3P                  
  nscm   = 1000,                 ! MD frequency to remove C.o.M motion         
  dt     = 0.002,                ! MD timestep                                 
  nrespa = 1,                    ! MD RESPA time step.  Keep nrespa * dt <= 4  
!  nstlim = 50000000,             ! MD Number of time steps                    
 nstlim = 1000,             ! MD Number of time steps                          
  tempi  = 310.0,                ! TEMP initial temperature                    
  temp0  = 310.0,                ! TEMP final temperature                      
  ntt    = 3,                    ! TEMP Temp control 3 = Langevin, 0 = NVE     
  gamma_ln = 5.0,                ! TEMP collision frequency                    
  ntf    = 2,                    ! POTENTIAL omit H-atoms in potential evaluati
  ntb    = 1,                    ! POTENTIAL Constant Volume PBC for non-bonded
  cut    = 12.0,                 ! POTENTIAL Non-bonded cutoff                 
  fswitch = 10.0,                ! POTENTIAL Switching function start length   
/                                                                              
                                                                               


Note: ig = -1. Setting random seed to   838870 based on wallclock time in 
      microseconds.
 
|--------------------- INFORMATION ----------------------
| GPU (CUDA) Version of PMEMD in use: NVIDIA GPU IN USE.
|                    Version 16.0.0
| 
|                      02/25/2016
| 
| Implementation by:
|                    Ross C. Walker     (SDSC)
|                    Scott Le Grand     (nVIDIA)
| 
| Precision model in use:
|      [SPFP] - Single Precision Forces, 64-bit Fixed Point
|               Accumulation. (Default)
| 
|--------------------------------------------------------
 
|----------------- CITATION INFORMATION -----------------
|
|    When publishing work that utilized the CUDA version
|    of AMBER, please cite the following in addition to
|    the regular AMBER citations:
|
|  - Romelia Salomon-Ferrer; Andreas W. Goetz; Duncan
|    Poole; Scott Le Grand; Ross C. Walker "Routine
|    microsecond molecular dynamics simulations with
|    AMBER - Part II: Particle Mesh Ewald", J. Chem.
|    Theory Comput., 2013, 9 (9), pp3878-3888,
|    DOI: 10.1021/ct400314y.
|
|  - Andreas W. Goetz; Mark J. Williamson; Dong Xu;
|    Duncan Poole; Scott Le Grand; Ross C. Walker
|    "Routine microsecond molecular dynamics simulations
|    with AMBER - Part I: Generalized Born", J. Chem.
|    Theory Comput., 2012, 8 (5), pp1542-1555.
|
|  - Scott Le Grand; Andreas W. Goetz; Ross C. Walker
|    "SPFP: Speed without compromise - a mixed precision
|    model for GPU accelerated molecular dynamics
|    simulations.", Comp. Phys. Comm., 2013, 184
|    pp374-380, DOI: 10.1016/j.cpc.2012.09.022
|
|--------------------------------------------------------
 
|------------------- GPU DEVICE INFO --------------------
|
|            CUDA_VISIBLE_DEVICES: not set
|   CUDA Capable Devices Detected:      1
|           CUDA Device ID in use:      0
|                CUDA Device Name: Tesla K20m
|     CUDA Device Global Mem Size:   4799 MB
| CUDA Device Num Multiprocessors:     13
|           CUDA Device Core Freq:   0.71 GHz
|
|--------------------------------------------------------
 
 
| Conditional Compilation Defines Used:
| PUBFFT
| BINTRAJ
| CUDA
| EMIL

| Largest sphere to fit in unit cell has radius =    44.450

| New format PARM file being parsed.
| Version =    1.000 Date = 04/03/17 Time = 21:47:27
| Force field information read from topology file: 
|              >>>> CHARMM36 All-Hydrogen Parameter File for Proteins <<<<<<<<<<  
|
|CHARMM: CHARMM force field in use. 

| Note: 1-4 EEL scale factors are being read from the topology file.

| Note: 1-4 VDW scale factors are being read from the topology file.
| INFO: Off Diagonal (NBFIX) LJ terms found in prmtop file.
|       The prmtop file has been modified to support atom
|       type based pairwise Lennard-Jones terms.
| Duplicated    0 dihedrals

| Duplicated    0 dihedrals

--------------------------------------------------------------------------------
   1.  RESOURCE   USE: 
--------------------------------------------------------------------------------

 getting new box info from bottom of inpcrd
 NATOM  =  115183 NTYPES =      45 NBONH =  107615 MBONA  =    7567
 NTHETH =   49571 MTHETA =   10290 NPHIH =   26934 MPHIA  =   18893
 NHPARM =       0 NPARM  =       0 NNB   =  213960 NRES   =   34667
 NBONA  =    7567 NTHETA =   10290 NPHIA =   18893 NUMBND =      84
 NUMANG =     194 NPTRA  =     454 NATYP =       1 NPHB   =       0
 IFBOX  =       1 NMXRS  =      45 IFCAP =       0 NEXTRA =       0
 NCOPY  =       0

|CHARMM: Reticulating splines.

| Coordinate Index Table dimensions:    19   15   13
| Direct force subcell size =     6.5198    6.8766    6.8384

     BOX TYPE: RECTILINEAR

--------------------------------------------------------------------------------
   2.  CONTROL  DATA  FOR  THE  RUN
--------------------------------------------------------------------------------

                                                                                

General flags:
     imin    =       0, nmropt  =       0

Nature and format of input:
     ntx     =       5, irest   =       1, ntrx    =       1

Nature and format of output:
     ntxo    =       2, ntpr    =    5000, ntrx    =       1, ntwr    =   50000
     iwrap   =       1, ntwx    =    5000, ntwv    =       0, ntwe    =       0
     ioutfm  =       1, ntwprt  =       0, idecomp =       0, rbornstat=      0

Potential function:
     ntf     =       2, ntb     =       1, igb     =       0, nsnb    =      25
     ipol    =       0, gbsa    =       0, iesp    =       0
     dielc   =   1.00000, cut     =  12.00000, intdiel =   1.00000

Frozen or restrained atoms:
     ibelly  =       0, ntr     =       0

Molecular dynamics:
     nstlim  =      1000, nscm    =      1000, nrespa  =         1
     t       =   0.00000, dt      =   0.00200, vlimit  =  -1.00000

Langevin dynamics temperature regulation:
     ig      =  838870
     temp0   = 310.00000, tempi   = 310.00000, gamma_ln=   5.00000

SHAKE:
     ntc     =       2, jfastw  =       0
     tol     =   0.00001

| Intermolecular bonds treatment:
|     no_intermolecular_bonds =       1

| Energy averages sample interval:
|     ene_avg_sampling =    5000

Ewald parameters:
     verbose =       0, ew_type =       0, nbflag  =       1, use_pme =       1
     vdwmeth =       0, eedmeth =       1, netfrc  =       1
     Box X =  123.876   Box Y =  103.149   Box Z =   88.900
     Alpha =   90.000   Beta  =   90.000   Gamma =   90.000
     NFFT1 =  128       NFFT2 =  108       NFFT3 =   96
     Cutoff=   12.000   Tol   =0.100E-04
     Ewald Coefficient =  0.22664
     Interpolation order =    4

--------------------------------------------------------------------------------
   3.  ATOMIC COORDINATES AND VELOCITIES
--------------------------------------------------------------------------------

                                                                                
 begin time read from input coords =  1070.000 ps

 
 Number of triangulated 3-point waters found:    33521

     Sum of charges from parm topology file =  -0.00000000
     Forcing neutrality...

| Dynamic Memory, Types Used:
| Reals             3079032
| Integers          4125735

| Nonbonded Pairs Initial Allocation:    57283385

| GPU memory information (estimate):
| KB of GPU memory in use:    912866
| KB of CPU memory in use:    170366

--------------------------------------------------------------------------------
   4.  RESULTS
--------------------------------------------------------------------------------

 ---------------------------------------------------
 APPROXIMATING switch and d/dx switch using CUBIC SPLINE INTERPOLATION
 using   5000.0 points per unit in tabled values
 TESTING RELATIVE ERROR over r ranging from 0.0 to cutoff
| CHECK switch(x): max rel err =   0.2738E-14   at   2.422500
| CHECK d/dx switch(x): max rel err =   0.7967E-11   at   2.716640
 ---------------------------------------------------
|---------------------------------------------------
| APPROXIMATING direct energy using CUBIC SPLINE INTERPOLATION
|  with   50.0 points per unit in tabled values
| Relative Error Limit not exceeded for r .gt.   2.27
| APPROXIMATING direct force using CUBIC SPLINE INTERPOLATION
|  with   50.0 points per unit in tabled values
| Relative Error Limit not exceeded for r .gt.   2.78
|---------------------------------------------------
wrapping first mol.:      123.87596      103.14935       88.89963
--------------------------------------------------------------------------------
   5.  TIMINGS
--------------------------------------------------------------------------------

|  NonSetup CPU Time in Major Routines:
|
|     Routine           Sec        %
|     ------------------------------
|     Nonbond           0.25    1.51
|     Bond              0.00    0.00
|     Angle             0.00    0.00
|     Dihedral          0.00    0.00
|     Shake             0.01    0.06
|     RunMD            16.04   98.42
|     Other             0.00    0.01
|     ------------------------------
|     Total            16.29

|  PME Nonbond Pairlist CPU Time:
|
|     Routine              Sec        %
|     ---------------------------------
|     Set Up Cit           0.00    0.00
|     Build List           0.00    0.00
|     ---------------------------------
|     Total                0.00    0.00

|  PME Direct Force CPU Time:
|
|     Routine              Sec        %
|     ---------------------------------
|     NonBonded Calc       0.00    0.00
|     Exclude Masked       0.00    0.00
|     Other                0.00    0.01
|     ---------------------------------
|     Total                0.00    0.01

|  PME Reciprocal Force CPU Time:
|
|     Routine              Sec        %
|     ---------------------------------
|     1D bspline           0.00    0.00
|     Grid Charges         0.00    0.00
|     Scalar Sum           0.00    0.00
|     Gradient Sum         0.00    0.00
|     FFT                  0.00    0.00
|     ---------------------------------
|     Total                0.00    0.00

|  Final Performance Info:
|     -----------------------------------------------------
|     Average timings for last    1000 steps:
|     Elapsed(s) =      16.35 Per Step(ms) =      16.35
|         ns/day =      10.57   seconds/ns =    8177.29
|
|     Average timings for all steps:
|     Elapsed(s) =      16.35 Per Step(ms) =      16.35
|         ns/day =      10.57   seconds/ns =    8177.29
|     -----------------------------------------------------

|  Setup CPU time:            2.32 seconds
|  NonSetup CPU time:        16.29 seconds
|  Total CPU time:           18.62 seconds     0.01 hours

|  Setup wall time:           4    seconds
|  NonSetup wall time:       16    seconds
|  Total wall time:          20    seconds     0.01 hours
